### PR TITLE
Added checkbox to allow preventing image embedding in HTML emails

### PR DESCRIPTION
**Pull request by Loilo**
Tested on different configurations, worked nicely so far.

Translations are mostly copy-pasted from the newsletter module with tiny adjustmens. I can't guarantee that Polish and Russion versions are exactly correct but I assume. 😉 

I'm a little bit uncertain about the naming of the added `useExternalImages()` method on the `EmailMessageDraft` class. It's the only one not following the `getSomething()` naming convention but it's also the only one returning a boolean value in there.
I found the "non-getty" version more precise and intuitive but feel free to change that if you insist on the `get` naimg pattern.

### CODE REVIEW

File: library/NotificationCenter/MessageDraft/EmailMessageDraft.php [hunk 1]
```diff
@@ -170,6 +170,14 @@ public function getHtmlBody()
     }
 
     /**
+     * Returns true if images should not be embedded
+     * @return boolean
+     */
+    public function useExternalImages() {
+        return !!$this->objLanguage->email_external_images;
```
Comment (Toflar) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: Why is there a `!!` if I may ask?

File: languages/ru/tl_nc_language.php [hunk 2]
```diff
@@ -25,6 +25,8 @@
 $GLOBALS['TL_LANG']['tl_nc_language']['edit']['1']                 = 'Редактировать язык ID %s.';
 $GLOBALS['TL_LANG']['tl_nc_language']['email_html']['0']           = 'HTML';
 $GLOBALS['TL_LANG']['tl_nc_language']['email_html']['1']           = 'Введите HTML.';
+$GLOBALS['TL_LANG']['tl_nc_language']['email_external_images']['0']= 'Внешние изображения';
+$GLOBALS['TL_LANG']['tl_nc_language']['email_external_images']['1']= 'Не вставлять изображения в HTML письма.';
```
Comment (Toflar) [inline languages/ru/tl_nc_language.php]: Please do not add any translations except for English. Translations are managed using transifex :)

File: library/NotificationCenter/MessageDraft/EmailMessageDraft.php (hunk 1 above)
Comment (aschempp) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: That's a force-boolean-casting 😎

File: languages/ru/tl_nc_language.php (hunk 2 above)
Comment (Loilo) [inline languages/ru/tl_nc_language.php]: Seems reasonable. :)

File: library/NotificationCenter/MessageDraft/EmailMessageDraft.php (hunk 1 above)
Comment (Toflar) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: That's to much JS for me. Please use `(bool)` as in the rest of the code :)

Comment (Loilo) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: I wanted this to return a boolean, not "" or "1".
Not that you ask I'm not even sure anymore if Contao does this conversion from CHAR(1) to boolean on its own. :)

Comment (Loilo) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: Should I attach a new pull request or are you gonna make the changes?
(Sorry for asking dumb questions btw, I'm actually pretty new to contributing to foreign repos. :blush:)

Comment (Toflar) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: Just push the changes to the branch ('optional-image-embedding') of your fork, the PR will automatically get updated ;-)

Comment (Loilo) [inline library/NotificationCenter/MessageDraft/EmailMessageDraft.php]: Great. 👍

**Comment by Loilo** [comment]
Always a pleasure. Thanks for the gentle introduction into contributing on GitHub. :wink:

**Comment by Toflar** [comment]
wtf, wanted to merge this in the first place. sorry for the confusion :D

**Pull request merged**